data-processing/common/commands/base.py

```diff
@@ -136,7 +136,9 @@ def read_arxiv_ids_from_file(path: Path) -> List[ArxivId]:
         for l in arxiv_ids_file:
             comment_start = l.find("#")
             if comment_start != -1:
-                arxiv_ids.append(l[:comment_start].strip())
+                arxiv_id_or_nothing = l[:comment_start].strip()
+                if arxiv_id_or_nothing != "":
+                    arxiv_ids.append(arxiv_id_or_nothing)
```

Comment:
Extra small fix unrelated to this PR, where when the pipeline is getting run based on a list of arXiv IDs in a file, individual arXiv IDs can be commented out.